Update.go

```diff
@@ -0,0 +1,48 @@
+func update(client *http.Client) {
+	fmt.Println("Starting program successfully... Output will be displayed only when records are updated.")
+	//Infinite loop to update records over time
+	for {
+		//GETS current record information
+		url := "https://api.cloudflare.com/client/v4/zones/"+os.Getenv("zoneId")+"/dns_records/"+os.Getenv("recordId")
```

Comment:
/dns_records/ will return only a single entity, you might need a list of records